03-Mains/Temperature_Salinity_Diagram.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "65fe6a4e-f751-40c3-80ed-12c66e728784",
```

Comment:
<p>Probably worth mentioning that this step causes lots of memory to get filled up, and could be a point of failure if people change the recipe to load in much more data because the kernel will die if you smash the memory limits.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>